web_external/Featuresets/FeaturesetsView.js

```diff
@@ -24,6 +24,11 @@ isic.views.FeaturesetsView = isic.View.extend({
         }, this);
         this.featuresets.fetch();
 
+        // TODO: Use the more general 'update' event, once Backbone is upgraded from Girder
+        this.listenTo(this.featuresets, 'remove', function () {
+            this.render();
+        }, this);
```

Comment:
Note that in this case it can be simplified further--just use `this.render()` as the third argument.